File: school/28-01-22/2.py
n = int(input("Enter the number of students: "))
stu = {}
for i in range(n):
    name = input("Enter the name of the student: ")
    roll_no = int(input("Enter the roll number of the student: "))
    marks = int(input("Enter the marks of the student: "))
    stu[roll_no] = {'name':name,'marks':marks}
stu_n = stu.copy()
for i in stu:
    if stu[i]['marks'] >= 75:
        pass
    else:
        del stu_n[i]
for i in stu_n:
    print(f"{i}:")
    for z in stu_n[i]:
        print(f"  {z}: {stu_n[i][z]}")

File: college/C/lab 12/2.c
#include<stdio.h>
struct stu{
    char student_name[100];
    int roll_no;
    float marks;
} s[2];
int main(){
    int i;
    for(i=0;i<2;i++){
        fflush(stdin);
        printf("Enter name: ");
        gets(s[i].student_name);
        printf("Enter roll no: ");
        scanf("%d",&s[i].roll_no);
        printf("Enter marks: ");
        scanf("%f",&s[i].marks);
    }
    for(i=0;i<2;i++){
        printf("\nName: %s\n",s[i].student_name);
        printf("Roll no: %d\n",s[i].roll_no);
        printf("Marks: %f\n",s[i].marks);
    }
    return 0;
}